src/App.tsx

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+
+import { actionsTodos } from './features/todos';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actionsTodo } from './features/currentTodo';
 
 export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const todos = useAppSelector(state => state.todos);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  useEffect(() => {
+    if (localStorage.getItem('todos') !== null) {
+      dispatch(actionsTodos.setTodos(JSON.parse(localStorage.getItem('todos') || '')));
+    } else {
```

Comment:
you can ignore this else
```suggestion
    }
```